raw_lidar_topic_name: /points_raw

lidar_cluster_topic_name: /perception/lidar_cluster

node_rate: 10   # [Herz]
